stories/SQFormDropdown.stories.js

```diff
@@ -37,7 +37,10 @@ const defaultArgs = {
 
 export const SQFormDropdown = args => (
   <div style={{minWidth: 250}}>
-    <SQFormStoryWrapper initialValues={{[defaultArgs.name]: ''}}>
+    <SQFormStoryWrapper
+      initialValues={{[defaultArgs.name]: ''}}
+      validationSchema={args.validationSchema}
```

Comment:
This should be destructured from `args`, otherwise it will be passed into the dropdown component on line 47. So
```js
const {validationSchema, ...dropdownProps} = args;
```
then you can spread `dropdownProps` on the dropdown component instead of `args`